pages/galleri/index.tsx

```diff
@@ -0,0 +1,228 @@
+import Layout, { Metadata } from '../../components/Layout';
+import styled from 'styled-components';
+import { useCallback, useEffect, useState } from 'react';
+import Image from 'next/image';
+import { itdageneBlue } from '../../utils/colors';
+
+const images = [
+  'DSC_1043.jpg',
+  'DSC_1003.jpg',
+  'DSC_1053.jpg',
+  'DSC_1105.jpg',
+  'DSC_1128.jpg',
+  'DSC_1137.jpg',
+  'DSC_1141.jpg',
+  'DSC_1145.jpg',
+];
+
+const GalleryWrap = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  align-items: center;
+  justify-content: center;
+  max-width: 1120px;
+  margin: 0 auto 60px;
+`;
+
+const SliderWrap = styled('div')`
```

Comment:
Just a note that react-modal is a thing that exists.

Reply:
Noted. This work just fine though